### DISCUSSION TITLE
无法更改壁纸存储位置

### DISCUSSION THREAD

**Discussion by blackandwhiteke** [rocksdanister/lively#2348] [General]
壁纸存储位置打开之后，无法更改

**Comment by rocksdanister** [comment]
<img width="673" alt="image" src="https://github.com/user-attachments/assets/da8323f0-c257-4970-b6dd-9ad451caaa9e">

(1) -> Open wallpaper install folder.
(2) -> Change wallpaper install folder.

It is confusing, I should reverse it 🤔.
